ui/src/dashboards/constants/index.ts

```diff
@@ -6,6 +6,10 @@ import {Cell, Dashboard, RemoteDataState} from 'src/types'
 import {DecimalPlaces} from 'src/types'
 import {DEFAULT_TIME_FORMAT} from 'src/shared/constants'
 
+import {SortTypes} from 'src/shared/utils/sort'
+import {Sort} from '@influxdata/clockface'
+import {DashboardSortKey} from 'src/shared/components/resource_sort_dropdown/generateSortItems'
```

Comment:
🤔 it's strange that this type is being set in a decentralized location. It might be beyond the scope of this PR to move this to `src/types` but it is worth considering doing that as a fast follow-up

Reply:
good suggestion, but I think it's beyond this PR's scope to do that.